ew/static/cfg.py

```diff
@@ -5298,8 +5314,8 @@
     "{} does a triple backflip! Hot diggedy!",
     "{} performs a double Cartwheel! Not really a dance move, but we'll take it!",
     "{} starts a Conga line! The party's over here!",
-    "{} does a moonwalk! They're smooth as heck!",
-    "{} does the robot! They manage to pull it off in a way that doesn't seem totally autistic!",
+    "{} does a moonwalk! They're smooth as heck! https://en.wikipedia.org/wiki/Moonwalk_(dance)",
+    "{} does the robot! They manage to pull it off in a way that doesn't seem totally idiotic!",
```

Comment:
quit trying to sand the edges off rfck's flavor text. you didn't write it originally and this is categorically an edgy place.  also, drop the wiki article, everybody knows what a moonwalk is and the visual aid is unwieldly

Reply:
Read all of these, radical. Understood completely 😌